### ISSUE TITLE
Bar Chart with open value shown incomplete with DateAxis

### ISSUE THREAD

**Issue by larstippmann** [amcharts/amcharts5#1139]
with  var day = new Date(2023, 9, 18); the series only show 4 of 5 DataItems, with   var day = new Date(2023, 9, 18, 1) it shows all DataItems. So a work around is there, but I think it's still a bug.

```HTML
<!DOCTYPE html>
<html>
<body>

<!-- Styles -->
<style>
  #chartdiv {
    width: 100%;
    height: 360px;
    border: 1px solid #aaa; 
    border-radius: 5px;
  }
  </style>
  
  <!-- Resources -->
  <script src="https://cdn.amcharts.com/lib/5/index.js"></script>
  <script src="https://cdn.amcharts.com/lib/5/xy.js"></script>
  
  <!-- Chart code -->
  <script>
  
  var colO = {
    fill: am5.color(0x169e49),
  }

  var colW = {
    fill: am5.color(0xedd633),
  }

  var day = new Date(2023, 9, 18);

  var data = [
  {
    "day": day.getTime(),
    "start": 0,
    "end": 20,
    "setting": colO,
  },
  {
    "day": day.getTime(),
    "start": 20,
    "end": 30,
    "setting": colW,
  },
  {
    "day": day.getTime(),
    "start": 30,
    "end": 40,
    "setting": colO,
  },
  {
    "day": day.getTime(),
    "start": 40,
    "end": 50,
    "setting": colW,
  },
  {
    "day": day.getTime(),
    "start": 50,
    "end": 60,
    "setting": colO,
  },
  ];
  
  am5.ready(function() {

  var root = am5.Root.new("chartdiv");

  // create chart
  var chart = root.container.children.push(am5xy.XYChart.new(root, {
    panX: false,
    panY: true,
    wheelX: "panX",
    wheelY: "zoomY",
    layout: root.verticalLayout,
  }));
  
  // create x and y axis
  var xAxis = chart.xAxes.push(am5xy.DateAxis.new(root, {
    baseInterval: { timeUnit: "day", count: 1 },
    renderer: am5xy.AxisRendererX.new(root, {})
  }));
  
  var yAxis = chart.yAxes.push(am5xy.ValueAxis.new(root, {
    min: 0,
    max: 60,
    renderer: am5xy.AxisRendererY.new(root, {})
  }));

  // create series
  var series = chart.series.push(am5xy.ColumnSeries.new(root, {
    xAxis: xAxis,
    yAxis: yAxis,
    openValueYField: "start",
    valueYField: "end",
    valueXField: "day",
    stroke: 0
  }));

  series.columns.template.setAll({
    templateField: "setting",
  });

  series.data.setAll(data);
  
  });
  </script>
  
  <!-- HTML -->
  <div style="padding:20px;">
    <div id="chartdiv"></div>
  </div>
</body>
</html>
```

**Comment by martynasma** [comment]
Repro CodePen:
https://codepen.io/team/amcharts/pen/abPeOdq/c6aee27f208659cdc817a04e794a3615?editors=0010

**Comment by martynasma** [comment]
Fixed in [5.5.1](https://github.com/amcharts/amcharts5/releases/tag/5.5.1).

## [5.5.1] - 2023-10-19

### Fixed
- Not all visual settings from columns were being copied to their legend markers.
- Data items with identical timestamps in data could sometimes be ignored when plotting series.
- `drawingsInteractive(false)` was not having an effect on some drawing tools (Vertical/Horizontal/Ray Line).
- `CategoryAxis`' elements (e.g. labels and grid) were ignoring `forceHidden` setting.
- Some line elements (e.g. grid) were being drawn somewhat blurry since `5.4.9`.

[Full change log](https://github.com/amcharts/amcharts5/blob/master/packages/shared/CHANGELOG.md).

[Download options](https://www.amcharts.com/download/).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.

**Comment by larstippmann** [comment]
Thanks a lot for the quick fix!

 

Lars

 

Von: Martynas Majeris ***@***.***> 
Gesendet: Donnerstag, 19. Oktober 2023 13:18
An: amcharts/amcharts5 ***@***.***>
Cc: larstippmann ***@***.***>; Author ***@***.***>
Betreff: Re: [amcharts/amcharts5] Bar Chart with open value shown incomplete with DateAxis (Issue #1139)

 

Fixed in 5.5.1 <https://github.com/amcharts/amcharts5/releases/tag/5.5.1> .


[5.5.1] - 2023-10-19


Fixed


*	Not all visual settings from columns were being copied to their legend markers.
*	Data items with identical timestamps in data could sometimes be ignored when plotting series.
*	drawingsInteractive(false) was not having an effect on some drawing tools (Vertical/Horizontal/Ray Line).
*	CategoryAxis' elements (e.g. labels and grid) were ignoring forceHidden setting.
*	Some line elements (e.g. grid) were being drawn somewhat blurry since 5.4.9.

Full change log <https://github.com/amcharts/amcharts5/blob/master/packages/shared/CHANGELOG.md> .

Download options <https://www.amcharts.com/download/> .

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
